### DISCUSSION TITLE
Support for ksqlDB

### DISCUSSION THREAD

**Discussion by ChenpiDog** [dbeaver/dbeaver#20659] [Q&A]
Refer to https://github.com/confluentinc/ksql.

**Comment by LonwoLonwo** [comment]
Hello @ChenpiDog 

We already added ksql support in this ticket: https://github.com/dbeaver/dbeaver/issues/3816
It is available only for PRO products in our Early Acces version now: https://dbeaver.com/files/ea/
And will be available in our next big PRO release 23.2.0 at 12th September 2023.
